### ISSUE TITLE
Go to Symbol in Workspace... does not show private methods or properties

### ISSUE THREAD

**Issue by kevinwhoffman** [bmewburn/intelephense#77]
## Expected Behavior

`Go to Symbol in Workspace...` should match any symbol in the workspace, including `private`, `protected`, and `public` methods and properties.

## Current Behavior

`Go to Symbol in Workspace...` does not show results for `private` methods or properties. Changing the visibility of the symbol to `protected` or `public` causes the symbol to appear in the results as expected.

`Go to Symbol in File...` works as expected and _does_ show results for `private` methods and properties, so the issue seems limited to the workspace-wide searches.

**Comment by jeffochoa** [comment]
I'm having the same issue, (using ubuntu)

**Comment by carloshhr** [comment]
Same issue here

**Comment by bmewburn** [comment]
This was as designed as I thought users would only use this for searching public symbols, but no reason why it cant be changed to include all symbols regardless of visibility.

**Comment by kevinwhoffman** [comment]
@bmewburn Thanks for considering it. I think most developers expect to be able to navigate to a symbol regardless of its visibility.

**Comment by jeffochoa** [comment]
Thanks! It would be nice for cases like :

```php

$this->foo = new FooClass;

// and go to the definition from here :

$this->foo->barMethod() ; // to FooClass@barMethod

```

**Comment by bmewburn** [comment]
This will be available in v0.8.3.
@jeffochoa in your case this should work if you phpdoc your class properties and barMethod is public
